### PR TITLE
feat: update sm version locator

### PR DESCRIPTION
**Pull request by jor2**
### Description

update sm da to latest version.

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

update sm da to latest version.

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Comment by jor2** [comment]
````
 2024/06/26 00:11:50 Terraform apply | Error: Invalid function argument
 2024/06/26 00:11:50 Terraform apply | 
 2024/06/26 00:11:50 Terraform apply |   on .terraform/modules/create_profile_attachment/modules/attachment/main.tf line 28, in locals:
 2024/06/26 00:11:50 Terraform apply |   28:   validate_profile = lookup(local.profile_map, var.profile_name, null) == null ? tobool("Could not find a valid profile name ${var.profile_name} and matching version ${var.profile_version}") : true
 2024/06/26 00:11:50 Terraform apply |     ├────────────────
 2024/06/26 00:11:50 Terraform apply |     │ while calling tobool(v)
 2024/06/26 00:11:50 Terraform apply |     │ var.profile_name is "CIS IBM Cloud Foundations Benchmark"
 2024/06/26 00:11:50 Terraform apply |     │ var.profile_version is "latest"
 2024/06/26 00:11:50 Terraform apply | 
 2024/06/26 00:11:50 Terraform apply | Invalid value for "v" parameter: cannot convert "Could not find a valid
 2024/06/26 00:11:50 Terraform apply | profile name CIS IBM Cloud Foundations Benchmark and matching version latest"
 2024/06/26 00:11:50 Terraform apply | to bool; only the strings "true" or "false" are allowed.
````

File: ibm_catalog.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
 						"authority": "scc-v3",
 						"profiles": [
 							{
-								"profile_name": "CIS IBM Cloud Foundations Benchmark",
+								"profile_name": "CIS IBM Cloud Foundations Benchmark v1.1.0",
 								"profile_version": "1.0.0"
```
Comment (ocofaigh) [inline ibm_catalog.json:73]: `profile_version` needs to be  `1.1.0`

**Review by ocofaigh — CHANGES_REQUESTED** [review]
1 comment

**Comment by ocofaigh** [comment]
@jor2 can you address the comment please - hoping to include this in a new release to prod next week

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**